src/assets/cs-guide.md

```diff
@@ -27,6 +27,43 @@ Please add yourself to this list (link is optional) if you've contributed to thi
 - [Dylan Souvage](https://github.com/firefelix)
 - [Kindeep Singh Kargil](https://kindeep.me)
 
+# Duration Types (D3 vs D2 etc.)
+
+D1 is a full year course, starting from Fall -> Winter, taking up two semesters.
+
+D2 is a one semester course taking place in the Fall, from September -> December.
+
+D3 is a one semester course taking place in the Winter, from January -> April.
+
+# Section Types
+
+The only difference is some classes are large and require splitting into multiple sections of a class for marking purposes and for seminar/lab purposes. If you for example want to be in a class with a friend make sure you enroll in the same section (this isn't always true, some classes don't matter what section you're in).
+
+# Common Course Types (2P00 VS 2F00)
+
+Note: 1 credit is equivalent to two classes, and every Major program requires 20 credits to graduate (40 classes)
+
+F - 1.00 Credit 
+P - 0.5 Credit 
+Q - 0.5 Credit
+N - 0 Credit (only required for co-op)
+C - 0 Credit (only required for co-op)
```

Comment:
`C` _technically_ counts for 0.5 credits:
<img width="748" alt="Screen Shot 2020-07-14 at 11 13 03" src="https://user-images.githubusercontent.com/13542746/87443064-0320d780-c5c3-11ea-8da5-1dea19806777.png">

Reply:
It's a little weird how it works. From what I know the weight is only there for OSAP purposes, but on Brock's side it doesn't count as a credit. (i.e. it doesn't speed up your graduation)